### PR TITLE
Add simple script to create new API versions

### PR DESCRIPTION
**Pull request by heaths**

### CODE REVIEW

**Comment by heaths** [comment]
OpenAPI Hub has been too unreliable, so I wanted to provide a script that would automate at least most of the work of creating a new API version for an existing service. We could spruce this script up in time to handle more scenarios.

File: eng/scripts/Copy-ApiVersion.ps1 [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+Write-Host ''
+Write-Host '!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!' -ForegroundColor 'Yellow'
+Write-Host '!!! IMPORTANT: Action Required !!!' -ForegroundColor 'Yellow'
+Write-Host '!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!' -ForegroundColor 'Yellow'
+Write-Host ''
+Write-Host "You must manually update the default version and copy $FromVersion sections from any markdown files in $readmeDirectory."
+Write-Host 'When complete, commit those changes as shown below, push to the upstream repo, and submit a pull request.'
+Write-Host ''
+Write-Host '    git commit -am"Updated readme files"'
```
Comment (benbp) [inline eng/scripts/Copy-ApiVersion.ps1:76]:
```suggestion
Write-Host '    git commit -am "Updated readme files"'
```

**Review by benbp — APPROVED** [review]

File: eng/scripts/Copy-ApiVersion.ps1 (hunk 1 above)
Comment (heaths) [reply eng/scripts/Copy-ApiVersion.ps1:76]: The space isn't needed, actually. That's always how I type it. Pretty common in posix command lines. If you really feel it necessary, though...

**Review by weshaggard — APPROVED** [review]

File: eng/scripts/Copy-ApiVersion.ps1 [hunk 2]
```diff
@@ -0,0 +1,77 @@
+[CmdletBinding()]
```
Comment (konrad-jamrozik) [inline eng/scripts/Copy-ApiVersion.ps1:1]: I would appreciate some kind of comment explaining why and when I would need/want to use this script.

Comment (maririos) [reply eng/scripts/Copy-ApiVersion.ps1:1]: Yes :) @JonathanCrd is going to take over this. When this happens we will make sure to update documentation and all of that.
Thank you @heaths !!!!

**Pull request merged**